app/org/sagebionetworks/bridge/services/email/ConsentEmailProvider.java

```diff
@@ -66,11 +67,9 @@ public MimeTypeEmail getMimeTypeEmail() throws MessagingException {
         final String sendFromEmail = String.format("%s <%s>", study.getName(), study.getSupportEmail());
         builder.withSender(sendFromEmail);
 
-        builder.withRecipient(user.getEmail());
-        Set<String> emailAddresses = BridgeUtils.commaListToSet(study.getConsentNotificationEmail());
-        for (String email : emailAddresses) {
-            builder.withRecipient(email);
-        }
+        Set<String> emailAddresses = Sets.newHashSet(BridgeUtils.commaListToSet(study.getConsentNotificationEmail()));
```

Comment:
Removing this. At some point I think the Set was immutable, but it's created every time you call that comma parsing method, and it's mutable in its final form.